app/ic.h

```diff
@@ -13,6 +13,9 @@ class Editor;
 
 class ICPrototype;
 
+//!
+//! \brief The IC class
+//!
```

Comment:
What's an IC? why it deserves a class?
the documentation should provide information for people (like me) that have no idea what the code is for.

Reply:
Oops, that's a Doxygen stub QtCreator made without my notice

I'll document `ic.h/.cpp` (and hopefully more), as well as address your previous comment in coming commits